README.md

```diff
@@ -1,4 +1,23 @@
+This program will call the hcship.DestroyLayer system call on the specified folder
 
-run file:
+This will effectively reclaim a lot of hard-drive space for an undocumented reason.
```

Comment:
Perhaps document it here.....   All the API actually does is take back-up privileges and effectively do the linux-equivalent to `rm -rf` or PowerShell `Remove-Item -Recurse -Force`. Also `a lot` is highly subjective.  Please be specific and say exactly what it is removing when used correctly (and potentially very dangerously).